server/src/main/java/org/opensearch/indices/store/IndicesStore.java

```diff
@@ -222,7 +220,7 @@ private void deleteShardIfExistElseWhere(ClusterState state, IndexShardRoutingTa
         String indexUUID = indexShardRoutingTable.shardId().getIndex().getUUID();
         ClusterName clusterName = state.getClusterName();
         for (ShardRouting shardRouting : indexShardRoutingTable) {
-            assert shardRouting.started() : "expected started shard but was " + shardRouting;
+            assert shardRouting.active() : "expected started shard but was " + shardRouting;
```

Comment:
assertion message should say - expected active shard?

Reply:
Updated it.